custom_reports/integrations/google_search_console.rst

```diff
@@ -0,0 +1,96 @@
+    +---------------------+---------------------------+----------+--------+--------------------------------------------------------------------------------------------------+
+    |Referrer URL         |referrer_url               |str       |False   |                                                                                                  |
+    +---------------------+---------------------------+----------+--------+--------------------------------------------------------------------------------------------------+
+    |Device type          |device_type                |[int, str]|True    |:download:`device_type.json </_static/json/enum/device_type.json>`                                |
+    +---------------------+---------------------------+----------+--------+--------------------------------------------------------------------------------------------------+
+    |Continent            |location_continent_iso_code|[str, str]|True    |:download:`location_continent_iso_code.json </_static/json/enum/location_continent_iso_code.json>`|
+    +---------------------+---------------------------+----------+--------+--------------------------------------------------------------------------------------------------+
+    |Country              |location_country_name      |[str, str]|True    |ISO 3166-2 codes (e.g. "PL")                                                                      |
+    +---------------------+---------------------------+----------+--------+--------------------------------------------------------------------------------------------------+
+    |Session entry URL    |session_entry_url          |str       |False   |                                                                                                  |
+    +---------------------+---------------------------+----------+--------+--------------------------------------------------------------------------------------------------+
+    |Timestamp            |timestamp                  |date      |False   |                                                                                                  |
+    +---------------------+---------------------------+----------+--------+--------------------------------------------------------------------------------------------------+
+    |Search engine keyword|search_engine_keyword      |str       |False   |                                                                                                  |
+    +---------------------+---------------------------+----------+--------+--------------------------------------------------------------------------------------------------+
+
+Mixed Queries
+-------------
+
+It is also possible to request both internal analytics and Google Search
+Console metrics in a single query (for example: "Sessions" and "Clicks (search
+engine)").
+
+However, **only the common dimensions listed below** may be used in such queries.
```

Comment:
I think this should be part of the previous sentence:
`... metrics in a single query (...), however **only the common dimensions listed below** may be used in such queries.`